pettingzoo/classic/rlcard_envs/texas_holdem_no_limit.py

```diff
@@ -151,7 +151,7 @@ def __init__(self, num_players=2, render_mode=None):
                                 ),
                                 [100, 100],
                             ),
-                            dtype=np.float64,
+                            dtype=np.float32,
```

Comment:
Is there a reason for this change? AFAICT, all environments tend to use float64 everywhere.

Reply:
It was failing due to the space and sampled action being different types somehow, not sure what the cause was but good to know, I’ll look into it and try to get it working with int64 instead